File: delivery/OrderPlaced.py

from AbstractEvent import AbstractEvent
import json

class OrderPlaced(AbstractEvent):
    id : int
    userId : str
    productId : str
    productName : str
    qty : int
    amount : int
    orderDt : 
    status : str
    address : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.userId = None
        self.productId = None
        self.productName = None
        self.qty = None
        self.amount = None
        self.orderDt = None
        self.status = None
        self.address = None

File: product/src/main/java/shopmyid/domain/OrderPlaced.java
package shopmyid.domain;

import java.util.*;
import lombok.*;
import shopmyid.domain.*;
import shopmyid.infra.AbstractEvent;

@Data
@ToString
public class OrderPlaced extends AbstractEvent {

    private Long id;
    private String userId;
    private String productId;
    private String productName;
    private Integer qty;
    private Integer amount;
    private Date orderDt;
    private String status;
    private String address;
}
